.github/workflows/focus-ios-ui-tests.yml

```diff
@@ -108,6 +112,13 @@ jobs:
                 name: ${{ env.browser }}-${{ matrix.xcodebuild_test_plan }}-${{ matrix.ios_simulator }}-xcodebuildlog-${{ github.run_number }}
                 path: ${{ env.browser }}/xcodebuild.log
                 retention-days: 90
+            - name: Upload junit files
```

Comment:
Let me have the junit file just in case of slack/github actions reporting issues in the future.